docs/mp/beanvalidation/02_general_beanvalidation.adoc

```diff
@@ -0,0 +1,71 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= General Bean Validation
+:h1Prefix: MP
+:pagename: general-bean-validation
+:description: General Bean Validation
+:keywords: helidon, webserver, bean validation, validation
+:bean-validation-spec-url: https://projects.eclipse.org/projects/ee4j.bean-validation
+:feature-name: General Bean Validation
+:common-deps-page-prefix-inc: ../../shared/dependencies/common_shared.adoc
+
+If BeanValidation is required outside JAX-RS/Jersey use cases, it is also available in Helidon.
+It follows the standard {bean-validation-spec-url}[Jakarta Bean
+Validation specification] which defines an API to validate Java beans.
+
+include::{common-deps-page-prefix-inc}[tag=maven-dependency]
+
+[source, xml]
+----
+<dependency>
+  <groupId>io.helidon.microprofile.bean-validation</groupId>
+  <artifactId>helidon-microprofile-bean-validation</artifactId>
+</dependency>
+----
+
+== Validation Example in Helidon MP
+
+The following example shows a simple application with one field declared as _not null_ using `@NotNull` annotation:
+
+[source,java]
+----
+public class GreetingHolder {
+    @NotNull
+    private String greeting;
+    //...
+}
+----
+
+Now, if in a bean there is a method parameter, annotated with `@Valid`, a _ValidationException_ will be thrown, if
+`GreetingHolder` with _null_ greeting is submitted:
+
+[source,java]
+----
+@ApplicationScoped
+public class GreetingProvider {
+    private GreetingHolder greetingHolder;
+    //..
+    void setGreeting(@Valid GreetingHolder greetingHolder) {
+        this.greetingHolder = greetingHolder;
+    }
+}
+----
+
+More information about validations supported is available can be found in {bean-validation-spec-url}[Jakarta Bean
+Validation specification].
```

Comment:
Perhaps the docs should remind users of `beans.xml` that is required to identify the beans and for BV to work.

Reply:
Added.